MotorController/index.rst

```diff
@@ -1,4 +1,10 @@
 Motor Controller
 ====
 
-Stephen better put stuff here
+The Motor Controller Subsystem is responsible for understanding how to wire the controller to the motor as well as interface with the controller through the configuration tool. In addition, this system develops a PCB that supplements the controller by adding features not supported by the controller.
+
+.. toctree::
+   :maxdepth: 1
```

Comment:
doesn't look like this is showing up in the build

Reply:
This `index` file needs to be added to the root `index` here: https://github.com/Solar-Gators/docs/blob/master/index.rst